fr/topics.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Topics-fr
```

Comment:
Do you plan to add french topics @Copinmalin ?

I dont believe we have tested translations of topics yet. So if you plan to do that in the near term, we can work on making sure that works. And if not in the short term, maybe we wait on topic changes? 

(Note: other translations do not have <language>/topics.md yet

Reply:
Hello Mike,

no I don't plan to add french topic for now, 
if necessary I can remove this folder

Reply:
Ok, can you remove for now? Thanks!